app/models/ta.rb

```diff
@@ -2,7 +2,9 @@ class Ta < User
 
   CSV_UPLOAD_ORDER = Rails.configuration.ta_csv_upload_order
   SESSION_TIMEOUT = Rails.configuration.ta_session_timeout
-
+  has_one :grader_permission, dependent: :destroy, foreign_key: :user_id
```

Comment:
add an `inverse_of :ta` here so that creating the associated object with nested params will work as intended